libbeat/publisher/pipeline/output_test.go

```diff
@@ -85,41 +85,51 @@ func TestMakeClientWorker(t *testing.T) {
 	}
 }
 
-func TestMakeClientWorkerAndClose(t *testing.T) {
+func TestReplaceClientWorker(t *testing.T) {
 	tests := map[string]func(mockPublishFn) outputs.Client{
 		"client":         newMockClient,
 		"network_client": newMockNetworkClient,
 	}
 
 	const minEventsInBatch = 50
+	const maxEventsInBatch = 150
 
 	for name, ctor := range tests {
 		t.Run(name, func(t *testing.T) {
 			seedPRNG(t)
 
 			err := quick.Check(func(i uint) bool {
-				numBatches := 1000 + (i % 100) // between 1000 and 1099
+				numBatches := 10000 + (i % 100) // between 1000 and 1099
```

Comment:
oops, I guess that was me. Want to to reduce it to `1000` again to reduce the running time?

Reply:
Reduced in 6e10331.